src/Kanvas/MappersImportersTemplates/Models/AttributeMapperImporterTemplate.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\MappersImportersTemplates\Models;
+
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+use Kanvas\Models\BaseModel;
+
+/**
+ * class AttributesImportersTemplates
+ * @property int $id
+ * @property int $importers_templates_id
+ * @property int $parent_id
+ * @property string $name
+ * @property string $mapping_field
+ *
+ * */
+class AttributeMapperImporterTemplate extends BaseModel
+{
+    protected $table = 'attributes_mappers_importers_templates';
```

Comment:
@FredPeal use the new tree library we are using in kanvas     use AsTree;
https://github.com/nevadskiy/laravel-tree